x/liquidity/types/liquidity.pb.go

```diff
@@ -141,6 +141,10 @@ type Pool struct {
 	ReserveAccountAddress string `protobuf:"bytes,4,opt,name=reserve_account_address,json=reserveAccountAddress,proto3" json:"reserve_account_address,omitempty" yaml:"reserve_account_address"`
 	// denom of pool coin of the pool
 	PoolCoinDenom string `protobuf:"bytes,5,opt,name=pool_coin_denom,json=poolCoinDenom,proto3" json:"pool_coin_denom,omitempty" yaml:"pool_coin_denom"`
+	// Swap fee rate for every executed swap
+	SwapFeeRate *github_com_cosmos_cosmos_sdk_types.Dec `protobuf:"bytes,10,opt,name=swap_fee_rate,json=swapFeeRate,proto3,customtype=github.com/cosmos/cosmos-sdk/types.Dec" json:"swap_fee_rate,omitempty" yaml:"swap_fee_rate"`
+	// Pool governor address - initially set to pool creator address - updated when someone deposit more funds
```

Comment:
```suggestion
	// Pool governor address - initially set to pool creator address, this address is updated when someone deposits more funds
```

Reply:
hi! Mostly this edit is to fix the subject/verb agreement, but we can also improve clarity with this little edit (apply same throughout, you'll see this suggestion in places this parameter is used). thanks!